server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/node/metedata/read/ChildPathsSchemaScanNode.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.mpp.plan.planner.plan.node.metedata.read;
+
+import org.apache.iotdb.commons.exception.IllegalPathException;
+import org.apache.iotdb.commons.path.PartialPath;
+import org.apache.iotdb.db.mpp.common.header.HeaderConstant;
+import org.apache.iotdb.db.mpp.plan.planner.plan.node.PlanNode;
+import org.apache.iotdb.db.mpp.plan.planner.plan.node.PlanNodeId;
+import org.apache.iotdb.db.mpp.plan.planner.plan.node.PlanNodeType;
+import org.apache.iotdb.tsfile.utils.ReadWriteIOUtils;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Objects;
+
+public class ChildPathsSchemaScanNode extends SchemaQueryScanNode {
+  // the path could be a prefix path with wildcard
+  private PartialPath prefixPath;
+
+  public ChildPathsSchemaScanNode(PlanNodeId id, PartialPath prefixPath) {
+    super(id);
+    this.prefixPath = prefixPath;
+  }
+
+  public PartialPath getPrefixPath() {
+    return prefixPath;
+  }
+
+  @Override
+  public PlanNode clone() {
+    return new ChildPathsSchemaScanNode(getPlanNodeId(), prefixPath);
+  }
+
+  @Override
+  public List<String> getOutputColumnNames() {
+    return HeaderConstant.showChildPathsHeader.getRespColumns();
+  }
+
+  @Override
+  protected void serializeAttributes(ByteBuffer byteBuffer) {
+    PlanNodeType.CHILD_PATHS_SCAN.serialize(byteBuffer);
+    ReadWriteIOUtils.write(prefixPath.getFullPath(), byteBuffer);
+  }
+
+  public static PlanNode deserialize(ByteBuffer buffer) {
+    String fullPath = ReadWriteIOUtils.readString(buffer);
+    PartialPath path;
+    try {
+      path = new PartialPath(fullPath);
+    } catch (IllegalPathException e) {
+      throw new IllegalArgumentException("Cannot deserialize ChildPathScanNode", e);
+    }
```

Comment:
We can use the serialize method in PartialPath and deserialize method in PathDeserializeUtil, which can avoid the exception try-catch.